frame/session/src/lib.rs

```diff
@@ -113,7 +113,6 @@
 //!
 //! ## Related Modules
 //!
-//! - [Consensus](../frame_consensus/index.html)
```

Comment:
There are more occurrences in this module. However, the whole description of the module in the docs is not correct anymore.

Reply:
(maybe not the whole, but crucial parts)